docs/modules/ROOT/pages/getting-started/relationships-and-complex-filtering.adoc

```diff
@@ -0,0 +1,235 @@
+= Relationships and complex filtering
+
+In a Graph database such as Neo4j, Relationships are as, if not more, important than Nodes. In this tutorial, you'll learn how to add relationships to the Cypher query from the xref:getting-started/filters-and-projections.adoc[previous tutorial]. 
+
+With relationships, often you'll need more complex filters such as the ones you'll add here. Finally, aliasing the results on the projection will make the results much easier to read and understand:
+
+```cypher
+MATCH(m:Movie)<-[r:ACTED_IN]-(a:Person)
+WHERE (a.name="Keanu Reeves" AND (NOT(m.title CONTAINS "Matrix") OR m.released < 2000))
+RETURN m.title, m.tagline, m.released, r.roles AS actingRoles
+```
+
+Previous queries were somewhat trivial, this one may deserve a bit more explanation. This query matches a Movie along with its actors, filtering movies with the actor `Keanu Reeves`. It also filters on either movies **not** containing `Matrix` or released before 2000. This query, using the default example Neo4j database, will effectively return the movies in which `Keanu Reeves` acted, except any Matrix movie after the first one, released in 1999.
+
+Finally, the projection also includes the first element of the relationship property `roles` (which is an array of the relationship). Because the column for this field will return `r.roles` as a name, an alias on this field to rename it to `actingRoles` will make the results more meaningful.
```

Comment:
```suggestion
Finally, the projection also includes the relationship property `roles`. Because the column for this field will return `r.roles` as a name, an alias on this field to rename it to `actingRoles` will make the results more meaningful.
```